config/main.py

```diff
@@ -533,7 +555,12 @@ def interface():
 def shutdown(interface_name, verbose):
     """Shut down interface"""
     if get_interface_mode() == "alias": 
-        interface_name = interface_alias(interface_name)
+        default_name = interface_alias_to_name(interface_name)
```

Comment:
Please refactor this code in the same manner as I suggested in my comment on line 411.